404.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>Error 404 - Page Not Found!</title>
+    <link rel="stylesheet" type="text/css" href="./assets/css/style.css" />
+    <link href="./assets/images/favicon.png" rel="shortcut icon" type="image/x-icon" />
+</head>
+
+<body>
+    <div class="container">
+        <img class="ops" src="./assets/images/404.svg" />
+        <br />
+        <h3>The Page you are looking for might have been removed 
```

Comment:
I believe there is a slight grammatical and spelling error here; shouldn't it be,

> The page you are looking for might have been removed, had its name changed, or its temporarily unavailable!